code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -404,12 +404,18 @@ var/const/NO_EMAG_ACT = -50
 	access = get_all_station_access() + get_centcom_access("Emergency Response Team")
 	..()
 
-/obj/item/weapon/card/id/legion
-	name = "\improper Tau Ceti Foreign Legion ID"
+/obj/item/weapon/card/id/distress
+	name = "\improper Freelancer Mercenary ID"
 	icon_state = "centcom"
-	overlay_state = "centcom"
+	assignment = "Freelancer Mercenary"
+
+/obj/item/weapon/card/id/distress/New()
+	access = list(access_legion, access_distress, access_maint_tunnels, access_external_airlocks, access_security, access_engine, access_engine_equip, access_medical, access_research, access_atmospherics, access_medical_equip)

```

Comment:
That is something I am unsure about for the Freelance Mercs.

If they are not associated with NT in any way, then why would have have access to a NT installation.

Reply:
I mean this is legion tier access. It's the bare minimum to get around maint.

Reply:
Well yes, but the legion is somewhat in bed with NT.
These mercs are neutral so they wouldnt (and shouldnt) have any access.

If they indeed start working for NT (which is not a given) then command can hand out access as needed.

Reply:
Solved